src/Symfony/Component/Serializer/Tests/Normalizer/ObjectNormalizerTest.php

```diff
@@ -740,6 +764,11 @@ class ObjectOuter
     private $inner;
     private $date;
 
+    /**
+     * @var ObjectInner[]
+     */
+    private $inners;
```

Comment:
if we're missing this phpdoc, the serializer won't try to normalize `inner` before calling `setInners()` right?